userspace/libsinsp/container.cpp

```diff
@@ -181,15 +181,21 @@ bool sinsp_container_engine_docker::parse_docker(sinsp_container_manager* manage
 		container->m_imageid = imgstr.substr(cpos + 1);
 	}
 
-	string hostname, port;
-	sinsp_utils::split_container_image(container->m_image,
-					   hostname,
-					   port,
-					   container->m_imagerepo,
-					   container->m_imagetag,
-					   container->m_imagedigest,
-					   false);
-	if(container->m_imagedigest.empty() || container->m_imagetag.empty())
+	bool no_name = strncmp(container->m_image.c_str(), container->m_imageid.c_str(),
+			       MIN(container->m_image.length(), container->m_image.length())) == 0;
```

Comment:
One of these should be container->m_imageid.length() right?

Reply:
Yup, will fix